src/lake/Lake/Config/LeanLib.lean

```diff
@@ -95,6 +95,15 @@ That is, the minimum of package's `buildType` and the library's  `buildType`.
   min self.pkg.buildType self.config.buildType
 
 /--
+The backend type for  modules of this library.
+The semantics are:
+1. If the package and the library don't specify one us the default (C for now)
+2. If the package specifices one but the library doesn't use that
+3. If the library specifices one use that regardless of the package
+-/
```

Comment:
This docstring initially confused me. For (1), `us` seems to be a typo for `use`. I also misread (2) because there was no comma between "doesn't" and "use". It is also rather wordy for a rather straightforward ordering. Perhaps the following is better:

```suggestion
/--
The backend type for modules of this library.
Prefer the library's `backend` configuration, then the package's, 
then the default (which is C for now).
-/
```